src/proxy-ui-api/frontend/package.json

```diff
@@ -12,7 +12,7 @@
     "test:coverage": "vue-cli-service test:unit --coverage",
     "check-audit": "check-audit",
     "resolve-audit": "resolve-audit",
-    "generate-types": "./node_modules/typescript/bin/tsc --esModuleInterop scripts/dtsgen-wrapper.ts && node scripts/dtsgen-wrapper.js",
+    "generate-types": "openapi --input ../src/main/resources/openapi-definition.yaml --output ./src/openapi-types",
```

Comment:
Seems to generate a lot of stuff we don't need (like mock server and such). Maybe add these flags to only generate the models `--exportCore false --exportServices false`

Reply:
Also the [serve and build scripts](https://github.com/nordic-institute/X-Road/blob/a0a783742352287bdfbc44e58f3a565c5834f8e2/src/proxy-ui-api/frontend/package.json#L6-L7) try to remove `src/openapi-types.ts`. Should probably be removing `src/openapi-types` folder.

Reply:
Yeah nicely spotted. I wasn't sure whether those core and services folders could turn out usefull.